src/depackers/stb_vorbis.c

```diff
@@ -3276,12 +3530,7 @@ static int vorbis_finish_frame(stb_vorbis *f, int len, int left, int right)
    if (f->previous_length) {
       int i,j, n = f->previous_length;
       float *w = get_window(f, n);
-
-      /* Sanity check */
-      if (w == NULL) {
-         return -1;
-      }
-
+      if (w == NULL) return 0;
```

Comment:
The mainstream version was added by https://github.com/nothings/stb/commit/98fdfc6df88b1e34a736d5e126e6c8139c8de1a6 but it doesn't fail like we do:
We rely on a negative return from `vorbis_finish_frame` to fail in our sanity checks.

`vorbis_pump_first_frame` is affected too: down below it returns 0 if it receives
non-zero from it, should  it not fail?

Reply:
I dug through the code a bit and I think any semantic difference between the upstream version and ours seems irrelevant here—`f->window[]` values should only be `NULL` if early stages of setup failed, weren't called, or if some memory was clobbered.